### PR TITLE
feat(snackbar): init package

### PR DESCRIPTION
**Pull request by soykje**
**TASK**: #1893

### Description, Motivation and Context
Init Snackbar. 

### Types of changes
- [x] ✨ New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: packages/components/snackbar/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    "aria",
+    "a11y",
+    "snackbar"
+  ],
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "build": "vite build"
+  },
```
Comment (acd02) [inline packages/components/snackbar/package.json:24]: Don't you want to add the `react-aria/spectrum` deps?

Comment (soykje) [reply packages/components/snackbar/package.json:24]: Why not, I thought I would do it later, but you're right :+1:

File: packages/components/snackbar/package.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  "dependencies": {
+    "@react-aria/toast": "3.0.0-beta.9",
+    "@spark-ui/icon": "^2.1.0",
+    "@spark-ui/icon-button": "^2.2.1",
+    "@spark-ui/icons": "^1.21.6",
```
Comment (soykje) [inline packages/components/snackbar/package.json:29]: @acd02 Also added these, as we'll need them at least for closing button.

**Review by acd02 — APPROVED** [review]

**Review by Powerplex — CHANGES_REQUESTED** [review]
Change version to 0.0.0 :)

**Comment by acd02** [comment]
well spotted @Powerplex 🤦, also don't forget to add the `experimental` prefix

**Review by Powerplex — APPROVED** [review]

**Pull request merged**